doc/api/esm.md

```diff
@@ -398,6 +402,9 @@ added:
   - v13.9.0
   - v12.16.2
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/00000
+    description: This property is no longer experimental.
```

Comment:
IMO we should first make `--experimental-import-meta-resolve` a no-op and expose an API to replace it. I have a PR for that, I just need to get back to it

Reply:
Second this. Can we make dirname and filename stable and leave resolve as-is til that's sorted?